nautobot/extras/views.py

```diff
@@ -1,5 +1,6 @@
 import inspect
 from datetime import datetime
+from celery import chain
```

Comment:
Please move this import to line 6 since it's 3rd party and should be alpha sorted for that section.